stats.go

```diff
@@ -107,3 +107,9 @@ func (s *statsExporter) submitMetric(v *view.View, row *view.Row, metricName str
 		return fmt.Errorf("aggregation %T is not supported", v.Aggregation)
 	}
 }
+
+func (s *statsExporter) stop() {
+	if err := s.client.Close(); err != nil {
+		s.opts.onError(err)
+	}
+}
```

Comment:
Could you add a simple test case for this function `stop()` ?

Just a simple test to assert that `onError` is called when `Close` returns an error.

Reply:
How? I don't know if there is a mocking setup in the tests?

Reply:
Something similar to this test that test errors should be ok: https://github.com/DataDog/opencensus-go-exporter-datadog/blob/188ce88f853d1bf5f0ab23ed0e5d1b3870518a49/stats_test.go#L77-L104